src/pages/part8.jsx

```diff
@@ -61,11 +61,14 @@ const Part8 = () => {
           Sign Up For Out Mailing List
         </Typography>
       {/* TODO: (3) Change the p tags to Typography with variant="body1" */}
+        <Typography variant="body1" alight="center">
         <p>Itaque earum rerum hic tenetur a sapiente delectus. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit.</p>
         <p>Non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem. At vero eos et accusamus. Facere possimus, omnis voluptas assumenda est, omnis dolor repellendus. Nisi ut aliquid ex ea commodi consequatur? Quis autem vel eum iure reprehenderit qui in ea voluptate velit esse quam.</p>
+        </Typography>
       {/* TODO: (4) Add a Link component. Ctrl+f for it in about.jsx to see how to use it.*/}
       {/* Make it link to your article. The title attribute is the text that appears when you hover over the link. */}
       {/* The Link component boils down to an <a> tag */}
+      <a href="http://localhost:8000/articles/2021/07/06/first-article">My first article</a>
```

Comment:
Use the imported `<Link>` component rather than an `<a>`. Link follows the theme of our website, whereas the plain tag doesn't.